main.go

```diff
@@ -82,9 +86,16 @@ func parseJSON(filename string) error {
 *	The main difference between types of loggers is the label (eg INFO) and additional data provided .
  */
 func initLogging() (*os.File, error) {
+	handlerEnv, err := settings.GetHandlerEnvironment()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to open handler environment")
+	}
+
+	logfile = handlerEnv.HandlerEnvironment.LogFolder + "/" + logfileLogName
```

Comment:
Use `path.join` instead of this. That would help take care of the Windows too whenever its enabled for it.
Eg: https://golang.org/pkg/path/#example_Join